docs/source/deployment/amazon_sagemaker.md

```diff
@@ -0,0 +1,9 @@
+# Amazon SageMaker
+
+Amazon SageMaker provides the components used for machine learning in a single toolset that supports both classical machine learning libraries like [`scikit-learn`](https://scikit-learn.org/) or [`XGBoost`](https://xgboost.readthedocs.io/), and Deep Learning frameworks such as [`TensorFlow`](https://www.tensorflow.org/) or [`PyTorch`](https://pytorch.org/).
+
+Amazon SageMaker is a fully-managed service and its features are covered by the [official service documentation](https://docs.aws.amazon.com/sagemaker/index.html).
+
+## The `kedro-sagemaker` plugin
+
+The `kedro-sagemaker` plugin from Get In Data | Part of Xebia enables you to run a Kedro pipeline on Amazon Sagemaker. Consult the [GitHub repository for `kedro-sagemaker`](https://github.com/getindata/kedro-sagemaker) for further details, or take a look at the [documentation](https://kedro-sagemaker.readthedocs.io/).
```

Comment:
`GetInData | Part of Xebia`

Reply:
What about video tutorial? Maybe you can embed it here? https://www.youtube.com/watch?v=yXIdz4kNMc8